### PR TITLE
Error handling and delay

### PR DESCRIPTION
**Pull request by Tackyou**
Delay is added because steam will block you for flooding if you request too fast

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -439,7 +453,9 @@
         };
 
         items.forEach(function(item) {
-            itemQueue.push(item);
+            setTimeout(function(){
+                itemQueue.push(item);
+            }, 500); // have some healthy delay or steam will block you for flooding
         });
```
Comment (tboothman) [inline main.js]: Surely all this does is delays adding anything to the queue for 500ms. I think the change of concurrency limit from 5 to 2 will have the effect you're after

Comment (Tackyou) [inline main.js]: I tried that but when pushing about 1000 trading cards to the market it still got me blocked. So I added this workaround which did what I wanted. If you don't like the delay, at least add the error handling which was really important. Without it the script would stop after some trading cards.

Comment (tboothman) [inline main.js]: Think about what exactly that items.foreach loop does
Before:
Add all items to list

After:
Start a setTimeout per item
wait 500ms
Add all items to the queue

Comment (Tackyou) [inline main.js]: It's already processing the queue while adding items to it, that's why it works.

**Comment by ylms** [comment]
Working great! If you could make remember prices for same items we would also get rid of blocking for price api. If that's possible.

**Pull request merged**